src/main/java/G.java

```diff
@@ -0,0 +1,166 @@
+import java.io.*;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class G {
+    private static final String FILE_PATH = System.getProperty("user.home") + "/Documents/CS2103T/tasks.txt";
+
+    public static void main(String[] args) {
+        String message = """
+                ____________________________________________________________
+                Hello! I'm G
+                What can I do for you?
+                ____________________________________________________________
+                """;
+
+        System.out.println(message);
+
+        ArrayList<Task> tasks = loadTasksFromFile(); // Load tasks from file
+        Scanner scn = new Scanner(System.in);
+        String str1 = scn.nextLine().trim();
```

Comment:
Consider renaming `str1` to a more descriptive variable name such as `userInput`.
```suggestion
        String userInput = scn.nextLine().trim();
```